packages/go/index.ts

```diff
@@ -694,9 +694,10 @@ Learn more: https://vercel.com/docs/runtimes#official-runtimes/go`
   }`;
 
   debug(`SPAWNING go build -o ${executable} ./... CWD=${tmp}`);
-  spawnSync('go', ['build', '-o', executable, './...'], {
+  execFileSync('go', ['build', '-o', executable, './...'], {
     cwd: tmp,
     env,
+    stdio: 'inherit',
```

Comment:
I think we only want stderr to print right? In the succesfull case we don't want to print anything?

Reply:
On successful builds, `go build` has no output.